packages/modal/src/components/modal/ModalFooter.tsx

```diff
@@ -0,0 +1,18 @@
+import { BoxProps, Column } from "@stenajs-webui/core";
+import * as React from "react";
+
+interface ModalFooterProps extends Omit<BoxProps, "indent" | "spacing"> {
+  sticky?: boolean;
+}
+
+export const ModalFooter: React.FC<ModalFooterProps> = (props) => (
+  <Column
+    indent={2}
+    spacing={2}
+    background={"var(--swui-modal-content-bg-color)"}
+    position={props.sticky ? undefined : "sticky"}
```

Comment:
```suggestion
    position={sticky ? undefined : "sticky"}
```

Also, is this correct? If `sticky` is true, it should be position=sticky, no?

Reply:
Inverted the naming to make it logical.